packages/dds/tree/src/simple-tree/api/schemaFactoryAlpha.ts

```diff
@@ -140,24 +137,24 @@ export class SchemaFactoryAlpha<
 	}
 
 	/**
-	 * {@inheritdoc schemaStatics.optionalRecursive}
-	 * @privateRemarks
-	 * The is only one of the many possible overrides for producing FieldSchemaAlpha.
-	 * If others are needed they can be desired.
+	 * {@inheritDoc SchemaStatics.optional}
 	 */
-	public override readonly optionalRecursive = <
-		const T extends ImplicitAllowedTypesUnsafe,
-		const TCustomMetadata = unknown,
-	>(
-		t: T,
-		props?: Omit<FieldProps<TCustomMetadata>, "defaultProvider">,
-	): FieldSchemaAlphaUnsafe<FieldKind.Optional, T, TCustomMetadata> => {
-		return createFieldSchemaUnsafe(FieldKind.Optional, t, props) as FieldSchemaAlphaUnsafe<
-			FieldKind.Optional,
-			T,
-			TCustomMetadata
-		>;
-	};
+	public static override readonly optional = schemaStatics.optional;
+
+	/**
+	 * {@inheritDoc SchemaStatics.required}
+	 */
+	public static override readonly required = schemaStatics.required;
+
+	/**
+	 * {@inheritDoc SchemaStatics.optionalRecursive}
+	 */
+	public static override readonly optionalRecursive = schemaStatics.optionalRecursive;
+
+	/**
+	 * Like {@link SchemaFactory.identifier} but static and a factory function that can be provided {@link FieldProps}.
+	 */
+	public static readonly identifier = schemaStatics.identifier;
```

Comment:
there used to be no way for public or even alpha API users to create an identifier field with a custom stored key, or metadata. Adding metadata support to generateSchemaFromSimpleSchema required adding this.